po/zh.po

```diff
@@ -1919,120 +1877,114 @@ msgstr "错误：无法将种子标记为已读: %s"
 #: rss/atomparser.cpp:19 rss/parser.cpp:374 rss/rss09xparser.cpp:21
 #: rss/rss10parser.cpp:20 rss/rss20parser.cpp:21
 msgid "XML root node is NULL"
-msgstr ""
+msgstr "XML 根节点是 NULL"
 
 #: rss/parser.cpp:258
 msgid "could not parse buffer"
 msgstr ""
 
 #: rss/parser.cpp:282
-#, fuzzy
 msgid "could not parse file"
-msgstr "错误：无法分析过滤器（filter）命令"
+msgstr "错误：无法解析文件"
 
 #: rss/parser.cpp:307
 msgid "no RSS version"
-msgstr ""
+msgstr "没有 RSS 版本"
 
 #: rss/parser.cpp:323
-#, fuzzy
 msgid "invalid RSS version"
-msgstr "无效位置！"
+msgstr "无效的 RSS 版本"
 
 #: rss/parser.cpp:344 rss/parser.cpp:355
-#, fuzzy
 msgid "invalid Atom version"
-msgstr "无效位置！"
+msgstr "无效的 Atom 版本"
 
 #: rss/parser.cpp:360
 msgid "no Atom version"
-msgstr ""
+msgstr "没有 Atom 版本"
 
 #: rss/rss09xparser.cpp:32
 msgid "no RSS channel found"
-msgstr ""
+msgstr "没有找到 RSS 频道"
 
 #: rss/rssparserfactory.cpp:31
 msgid "unsupported feed format"
-msgstr ""
+msgstr "不支持的种子格式"
 
 #: rust/libnewsboat/src/cliargsparser.rs:215
 msgid "%s: %s: invalid loglevel value"
-msgstr ""
+msgstr "%s: %s: 无效的日志等级"
 
 #: rust/libnewsboat/src/configpaths.rs:85
-#, fuzzy
 msgid ""
 "Fatal error: couldn't determine home directory!\n"
 "Please set the HOME environment variable or add a valid user for UID %u!"
-msgstr "请设置主目录的环境变量，或者添加一个有效的用户其UID为 %u!"
+msgstr "严重错误：无法确定家目录！\n"
+"请设置主目录的环境变量，或者添加一个有效的用户其UID为 %u!"
```

Comment:
I reckon the `HOME` variable should be included here, too. Since it's actually called `HOME`, it should not be translated.

Reply:
You're right. I missed all `HOME` at first, fixed some later but missed this one. Updated in the latest commit.